lib/client/session.go

```diff
@@ -270,7 +270,7 @@ func selectKeyAgent(tc *TeleportClient) agent.ExtendedAgent {
 	switch tc.ForwardAgent {
 	case ForwardAgentYes:
 		log.Debugf("Selecting system key agent.")
-		return tc.localAgent.systemAgent
```

Comment:
i guess base of PR should be `joerger/fix-agent-forwarding-without-local-agent-keys`